proto/resource.proto

```diff
@@ -27,5 +26,14 @@ message ResourceName {
   // The cache isolation type of this resource. For now, that can
   // be either CAS or AC. Other cache types may exist in the future.
   // Ex. CAS, AC
-  cache.CacheType cache_type = 4;
+  CacheType cache_type = 4;
+}
+
+// CacheType represents the type of cache being written to.
+enum CacheType {
```

Comment:
There will be an import cycle if we import the cache file. Once we finish the ResourceName refactor the cache package will probably frequently need to reference the resource package, so makes sense to me to prevent this here